src/frontend/src/components/plugins/PluginContext.tsx

```diff
@@ -1,18 +1,26 @@
-import { MantineColorScheme, MantineTheme } from '@mantine/core';
+import {
+  MantineColorScheme,
+  MantineTheme,
+  useMantineColorScheme,
+  useMantineTheme
+} from '@mantine/core';
 import { AxiosInstance } from 'axios';
-import { NavigateFunction } from 'react-router-dom';
+import { useMemo } from 'react';
+import { NavigateFunction, useNavigate } from 'react-router-dom';
 
-import { ModelType } from '../../enums/ModelType';
-import { SettingsStateProps } from '../../states/SettingsState';
-import { UserStateProps } from '../../states/UserState';
+import { api } from '../../App';
+import { useLocalState } from '../../states/LocalState';
+import {
+  SettingsStateProps,
+  useGlobalSettingsState,
+  useUserSettingsState
+} from '../../states/SettingsState';
+import { UserStateProps, useUserState } from '../../states/UserState';
 
-/*
+/**
```

Comment:
Q: Why is this second star needed?

Reply:
That's what the jsdoc spec says. (Ref: [jsdoc.app/about-getting-started](https://jsdoc.app/about-getting-started#:~:text=Each%20comment%20must%20start%20with%20a%20/**%20sequence%20in%20order%20to%20be%20recognized%20by%20the%20JSDoc%20parser.%20Comments%20beginning%20with%20/*%2C%20/***%2C%20or%20more%20than%203%20stars%20will%20be%20ignored.)) You can also see the difference in vscode, if there are two `**` vscode parses and highlights the jsdoc correctly.

Reply:
I was not aware of that, thank you for the explanation